.rubocop.yml

```
@@ -44,3 +44,11 @@ RSpec/ContextWording:
   Prefixes:
     - when
     - given
+
+RSpec/NestedGroups:
+  Max: 5
+
+RSpec/MultipleExpectations:
+  Max: 10
```

Ten seems rather high for most unit tests 😅 It is almost like having no limits (it is like disabling the cop, really). What are the use cases for this? IMO, 3-5 expectations should be be sufficient at most.

---

```suggestion
  Max: 5
```

I'm fine with 5 as well now that we have excluded the system tests from this rule 🙌 
the main uses of 10 were system tests.